plugins/dashboard/explorer_routes.go

```diff
@@ -99,7 +109,7 @@ func setupExplorerRoutes(routeGroup *echo.Group) {
 		search := c.Param("search")
 		result := &SearchResult{}
 
-		if len(search) < 81 {
+		if len(search) < 33 {
```

Comment:
Since addresses and messages have different lengths of IDs, there's actually no reason to spin up two goroutines to query for them, since the search parameter is for one or the other.

Also consider using the `message.IdLength` or `address.Length` respectively to check for the length and what action to take.